src/networks.json

```diff
@@ -1021,7 +1021,7 @@
     ],
     "explorer": "https://explorer.publicmint.io",
     "start": 19454413,
-    "logo": "ipfs://bafkreiglez3xnj3s63pgcky2znt5adyqrqjrrmxrcyhqitvlg2nikxewqi"
+    "logo": "ipfs://bafkreihh4y35xgshx7wbf7a2xj35do55f7lj4nhxqxtt5nqoohoojdvkki"
```

Comment:
Hey, @rafaeltorres-seegno This IPFS doesn't resolve. make sure to pin it with a third party provider

Reply:
hey @ChaituVR, you mean this? https://bafkreihh4y35xgshx7wbf7a2xj35do55f7lj4nhxqxtt5nqoohoojdvkki.ipfs.w3s.link/